addons/mail/static/src/models/message/message.js

```diff
@@ -418,6 +418,14 @@ function factory(dependencies) {
             );
         }
 
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsEmpty() {
+            return !this.body && this.attachments.length === 0;
```

Comment:
Supposing we keep this behavior (see other comments), I think this should also take into account message subtype description & tracking messages. I'm surprised no test is covering these, it should be red right now.

Isn't empty body actually `<p>` something also?